app/src/main/AndroidManifest.xml

```diff
@@ -57,9 +64,6 @@
             android:enabled="true"
             android:exported="true">
             <intent-filter>
-                <action android:name="android.intent.action.BOOT_COMPLETED" />
-                <action android:name="android.intent.action.QUICKBOOT_POWERON" />
```

Comment:
I ended up just removing QUICKBOOT_POWERON (both versions). They did not appear to actually do anything that I could see before.

Reply:
We added these extra actions because some device manufactures use different actions.
https://stackoverflow.com/a/42622385

Reply:
I should clarify. The LocationBroadcastReceiver appears to me to completely ignore these which is why I thought they could be deleted. It looks like it only processed BOOT_COMPLETED Honestly I am not sure if they are a good idea to include. 

**https://github.com/home-assistant/home-assistant-android/blob/f6618527d4f5f1233e7e0515f9816eb3ac282e5b/app/src/main/java/io/homeassistant/companion/android/background/LocationBroadcastReceiver.kt#L57**